email_notification/templates/email/html/partials/base.jinja

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="{{ language }}">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>{% block title %}Varaamo{% endblock %}</title>
+
+    <style>
+        @font-face {
+            font-family: "HelsinkiGrotesk";
+            src: url("https://makasiini.hel.ninja/delivery/HelsinkiGrotesk/565d73a693abe0776c801607ac28f0bf.woff") format("woff");
+        }
+
+        body {
+            background-color: #FFFFFF;
+            max-width: 960px;
+            margin: 0 auto;
+            font-family: HelsinkiGrotesk, serif;
+        }
+
+        p {
+            margin: 0;
+            font-size: 18px;
+        }
+
+        h2 {
+            font-size: 20px;
+            margin-top: 15px;
+            margin-bottom: 15px;
+        }
+
+        #header {
+            display: flex;
+            flex-direction: column;
+            gap: 50px;
+            padding: 30px 25px;
+            background-color: #FFFFFF;
+
+            .headerTopRow {
+                display: flex;
+                flex-direction: row;
+                align-items: center;
+                gap: 25px;
+
+                b {
+                    font-size: 24px;
+                }
+
+                img {
+                    width: 215px;
+                }
+            }
+
+            .headerBottomRow {
+                b {
+                    font-size: 36px;
+                }
+            }
+        }
+
+        #content {
+            display: flex;
+            flex-direction: column;
+            gap: 20px;
+            margin-bottom: 30px;
+
+            .main-container {
+                background-color: #F7F2E4;
+                padding: 20px 25px;
+
+                p {
+                    padding: 2px 0;
+                }
+            }
+
+            .secondary-container {
+                background-color: #FFFFFF;
+                padding: 0 25px;
+
+                p {
+                    font-size: 16px;
+                }
+            }
+        }
+
+        #footer {
+            display: flex;
+            flex-direction: column;
+            gap: 25px;
+            padding: 25px;
+            margin-top: 50px;
+            background-color: #E6E6E6;
+
+            .footerTopRow {
+                display: flex;
+                flex-direction: row;
+                align-items: center;
+                gap: 25px;
+
+                b {
+                    font-size: 16px;
+                }
+
+                img {
+                    width: 120px;
+                }
+            }
+
+            .footerBottomRow {
+                font-size: 14px;
+            }
+
+
+            .spacer-line {
+                border-top: #CCCCCC 2px solid;
```

Comment:
"unreliable" (maybe not much of a practical issue): https://www.caniemail.com/features/css-border/